packages/sources/svelte/src/components/indicators/VtmnRating/test/VtmnRating.spec.js

```diff
@@ -84,6 +91,42 @@ describe('VtmnRating', () => {
       const { container } = render(VtmnRating, { name: 'rating', value: 2 });
       expect(getCommentSecondary(container)).toBeUndefined();
     });
+
+    test('Should have rate article aria label for radio group', () => {
+      const { container } = render(VtmnRating, {
+        name: 'rating',
+        value: 2,
+        ariaLabel,
+      });
+      expect(getRating(container)).toHaveAttribute(
+        'aria-label',
+        '4 star out of 5',
+      );
+    });
+
+    test('Should have rate article aria label for radio group', () => {
+      const { getByRole } = render(VtmnRating, {
+        name: 'rating',
+        value: 2,
+        ariaLabel,
+      });
+      expect(
+        getByRole('radiogroup', { name: ariaLabel.rateArticle }),
+      ).toBeVisible();
+    });
```

Comment:
the last 2 tests have the same named and could be grouped together don't you think ?